test/sqlsmith/mzcompose.py

```diff
@@ -22,13 +22,8 @@
 )
 from materialize.mzcompose.services.materialized import Materialized
 
-if os.getenv("BUILDKITE_AGENT_META_DATA_AWS_INSTANCE_TYPE") == "c5.2xlarge":
```

Comment:
c5.2xlarge hasn't been in use for a while, so we used NUM_SERVERS = 1 (and the wrong memory limit)